BangazonAPI/Models/Computer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Computer
+    {
```

Comment:
The instructions to test say I should see a reflection of the database in here-- are these meant to be blank?

Reply:
the team is currently working to correct this issue caused by lack of AutoSave. We are installing this ext too, as we repopulate our Models. Thank you